.env.example

```
@@ -1,95 +1,169 @@
 SERVER_HOST=0.0.0.0
 SERVER_PORT=8010
-DEBUG=false
+DEBUG=true
 PRODUCTION=false
-
-# Logging Configuration
 LOG_LEVEL=INFO
 LOG_FORMAT=console
-LOG_TO_FILE=false
-# LOG_FILE=/var/log/crawlerr/server.log
+LOG_FILE=logs/crawlerr.log
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The LOG_FILE key should go before the LOG_FORMAT key

(UnorderedKey)